src/bin/pgcopydb/string_utils.c

```diff
@@ -744,11 +744,12 @@ pretty_print_bytes_per_second(char *buffer, size_t size, uint64_t bytes,
 		"PBit/s",                   /* Petabits per second */
 		"EBit/s"                    /* Exabits per second */
 	};
+	uint suffixes_len = sizeof(suffixes) / sizeof(suffixes[0]);
 
 	uint sIndex = 0;
 	long double count = ((long double) bytes) * 1000 * 8 / durationMs;
 
-	while (count >= 10000 && sIndex < 7)
+	while (count >= 10000 && sIndex < suffixes_len - 1)
```

Comment:
Same comment, make it so the human reader does NOT need to remember operators precedence rules.